portal-api/src/v1/facility-dates/cover-end-date.test.js

```diff
@@ -93,6 +93,20 @@ describe('updateCoverEndDate', () => {
     expect(Math.round(result.coverEndDate / 100)).toEqual(Math.round(coverEndDate / 100));
   });
 
+  // There should be tests for date and year but they don't return NaN!
```

Comment:
updateCoverEndDate has weird behaviour for invalid date and years because moment does

```js
 const moment = require('moment');

 const date1 = moment().set('date', NaN)
 const date2 = moment().set('month', NaN)
 const date3 = moment().set('year', NaN)
 
 console.log(moment(date1).format()); //  2024-01-31T16:49:39+00:00
 console.log(moment(date2).format()); //  Invalid date
 console.log(moment(date3).format()); //  2024-01-31T16:49:39+00:00
```

Reply:
I'll add tests here for now & possibly move them across later